patches/tModLoader/Terraria/ModLoader/Config/ModConfig.cs

```diff
@@ -19,6 +21,12 @@ public abstract class ModConfig
 	[JsonIgnore]
 	public string Name { get; internal set; }
 
+	[JsonIgnore]
+	public string LocalizationCategory => "Configs";
```

Comment:
Can't implement `ILocalizedModType` I suppose, hmm, cause they don't autoload. I think they should extend `ILocalizedModType` but just add `[Autoload(false)]` to `ModConfig`, should work